knowledge_base/Environment-Variables.md

```diff
@@ -221,12 +229,17 @@ If set to `e2e-test`, the app will bypass usage of real ETH RPCs and use the moc
 
 Owner: Ian Rowan
 
+## EVM_CE_POLL_INTERVAL
+
+Required for testing contests. Number of milliseconds. `10000` is recommended; if the number is too low, millions of requests will be made.
```

Comment:
It's not *required* necessarily since there's a default value. Also, it's not just for testing contests– it's for contests in general regardless of the environment.

Reply:
It's not just for contests. It's also for chain-event notifications. Should note that this env var will be deprecated soon.